src/lib/init.ts

```diff
@@ -331,12 +330,14 @@ export const initTSorJSProject = async ({
   } else {
     const TS_CONFIG = {
       compilerOptions: {
-        target: "es2016",
-        module: "commonjs",
-        esModuleInterop: true,
-        forceConsistentCasingInFileNames: true,
+        target: "es2022",
```

Comment:
Are you sure we should change to content of the JS template right now before we released the new version of JS Client?

Reply:
It works like this. Tests make sure it works too